go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/oras-project/oras-credentials-go
```

Comment:
Should it be `oras.land/oras-credentials-go`?

Reply:
@TerryHowe Let's make it consistent with `oras.land/oras-go`. We may need another issue to discuss if we need to change it to `github.com`.